### DISCUSSION TITLE
ABC contains local modifications!

### DISCUSSION THREAD

**Discussion by Simonliudan** [The-OpenROAD-Project/OpenROAD#3489] [Q&A]
[ 12%] Building abc/abc-bafd2a7
Debug: ABCREV=bafd2a7
[ 22%] Building passes/techmap/filterlib.o
[ 22%] Building yosys-smtbmc
ERROR: ABC contains local modifications! Set ABCREV=default in Yosys Makefile!
make: *** [Makefile:749: abc/abc-bafd2a7] Error 1

in Centos, I have set ABCREV = default, but the debug info is bafd2a7

**Comment by vijayank88** [comment]
Is this installation issue?
Can you provide steps to reproduce the same?

**Reply by Simonliudan** [reply]
git clone --recursive https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts
cd OpenROAD-flow-scripts
sudo ./setup.sh
./build_openroad.sh --local
the error occured

in the tools/yosys/Makefile, i set ABCREV = default, the error still there

**Comment by vvbandeira** [comment]
@Simonliudan, this looks more like an issue than a discussion. Please file an Issue and follow the Issue template, the template give us important information to help you.
